pkg/asset/tls/mcscertkey.go

```diff
@@ -3,14 +3,15 @@ package tls
 import (
 	"crypto/x509"
 	"crypto/x509/pkix"
-	"github.com/openshift/installer/pkg/types/ovirt"
 	"net"
 
 	"github.com/openshift/installer/pkg/asset"
 	"github.com/openshift/installer/pkg/asset/installconfig"
 	baremetaltypes "github.com/openshift/installer/pkg/types/baremetal"
 	openstacktypes "github.com/openshift/installer/pkg/types/openstack"
 	openstackdefaults "github.com/openshift/installer/pkg/types/openstack/defaults"
+	ovirttypes "github.com/openshift/installer/pkg/types/ovirt"
```

Comment:
Intentionally included here?

Reply:
I was standardizing the imports to use `*types`.  I can back that out if it is an issue.